spec/support/embeds_helpers.rb

```diff
@@ -1,11 +1,22 @@
 module EmbedsHelpers
-  def stub_head_request(url, status_code = 200)
-    stub_request(:head, url)
+  def stub_network_request(url:, method: :head, status_code: 200)
```

Comment:
I decided to condense the `head` and `get` calls I use in the specs into one method 🎉